CHANGES_UNRELEASED.md

```diff
@@ -21,4 +21,7 @@ Use the template below to make assigning a version number during the release cut
 ## Nimbus FML
 
 ### What's Changed
-  - Add `channels` value for defaults and add support for multiple channels in `channel` via comma separation. ([#5101](https://github.com/mozilla/application-services/pull/5101))
\ No newline at end of file
+  - Add `channels` value for defaults and add support for multiple channels in `channel` via comma separation. ([#5101](https://github.com/mozilla/application-services/pull/5101))
+
+### What's New
+  - JEXL targeting allows for using the `in` keyword with objects and a map of active experiments has been added to TargetingAttributes ([#5104](https://github.com/mozilla/application-services/pull/5104))
```

Comment:
Maybe a note that the map is currently always empty?

Reply:
Done!